### DISCUSSION TITLE
p/linux-tools-gource/

### DISCUSSION THREAD

**Discussion by giscus** [debuginn/blog#109] [General]
# p/linux-tools-gource/

Gource 是一款版本控制可视化的工具，使用这个工具可以将自己的 Git 提交的代码包括对 Mercurial，Bazaar 和 SVN 的内置日志生成可视化支持。Gou

https://blog.debuginn.com/p/linux-tools-gource/

**Comment by henry-2026** [comment]
请问下, 左侧的那个统计是什么生成的呢?

**Reply by debuginn** [reply]
图片左侧的代码行数统计？

**Reply by henry-2026** [reply]
嗯是的, 这个是怎么生成的呢

**Reply by debuginn** [reply]
这个应该是历史某个版本支持 我看现在不支持了 可以看下其他的开源统计工具

**Reply by lly-ke** [reply]
代码行数统计还有的  gource  --key
